src/pages/resources/hackathon-resources.mdx

```diff
@@ -41,12 +31,27 @@ Always refer back to the [getting help](/resources/getting-help) page for the la
 - [Hackathon Discord](https://discord.gg/h3AvTHj)
 - [Developer Discord](https://discord.gg/2YHSAey)
 
-## Tutorials
+## Documentation and tutorials
```

Comment:
```suggestion
## Tutorials
```

Reply:
Going back to comment below, is it necessary to include documentation here?

Reply:
Removed Documentation, which can be reached at the top navigation